distribution/pom.xml

```diff
@@ -171,7 +171,6 @@
 
     <modules>
         <module>fully-loaded</module>
```

Comment:
Totally agree. At the end of the day, I really think that elasticsearch will be more a container of plugins and every single will be a plugin which can be or not packaged within the final distribution or available as a plugin. Like what Logstash is today.

About optional deps, when embedding elasticsearch java client in an app, I had to add:

```xml
<dependency>
            <groupId>org.apache.lucene</groupId>
            <artifactId>lucene-expressions</artifactId>
            <version>5.2.1</version>
        </dependency>
```

Otherwise, the client complains about some missing parts...

May be this one should not be optional at the very least?